app/models/mission_control/items/page.rb

```diff
@@ -1,24 +1,24 @@
-class MissionControl::Jobs::Page
+class MissionControl::Items::Page
   DEFAULT_PAGE_SIZE = 10
 
-  attr_reader :jobs_relation, :index, :page_size
+  attr_reader :items_relation, :index, :page_size
```

Comment:
I suggest another name that matches other pagination gems elsewhere: _records_ instead of _items_.